### PR TITLE
Juggernaut

### PR DESCRIPTION
**Pull request by KingTerrytheTerrible**
## Changelog
:cl:
add: new traitor item - the Juggernaut helmet
/:cl:
makes you go fast, go through walls, be unstunnable and look stupid doing it

### CODE REVIEW

File: hippiestation/code/game/objects/items/juggernaut.dm [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    clothing_flags = THICKMATERIAL | STOPSPRESSUREDAMAGE | SHOWEROKAY
+    icon = 'hippiestation/icons/mob/head.dmi'
+    icon_state = "juggernaut_helmet"
+    item_state = "juggernaut_helmet"
+    slowdown = -1
+    alternate_worn_icon = 'hippiestation/icons/mob/head.dmi'
+    item_color = "juggernaut_helmet"
+    
+/obj/item/clothing/head/helmet/juggernaut/Initialize()
+    ADD_TRAIT(src, TRAIT_NODROP, JUGGERNAUT_TRAIT)
```
Comment (ghost) [inline hippiestation/code/game/objects/items/juggernaut.dm]: this should be in `equipped`

File: hippiestation/code/game/objects/items/juggernaut.dm [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+/obj/item/clothing/head/helmet/juggernaut/dropped()
+    qdel(src)
```
Comment (ghost) [inline hippiestation/code/game/objects/items/juggernaut.dm:14]: wouldn't `DROPDEL` make this redundant?

**Comment by steamp0rt** [comment]
+1 btw

**Comment by TrickyDickRichard** [comment]
how about some screenshots or a video or even just a picture of the sprite

File: code/__DEFINES/traits.dm [hunk 3]
```diff
@@ -235,3 +235,5 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
 #define HIVEMIND_ONE_MIND_TRAIT "one_mind"
 #define VR_ZONE_TRAIT "vr_zone_trait"
 #define MADE_UNCLONEABLE "made-uncloneable"
+/*Hippie*/
+#define JUGGERNAUT_TRAIT "juggernaut_trait"
```
Comment (ghost) [inline code/__DEFINES/traits.dm:239]: also this is fine but we have our own traits.dm in `~hippie-defines`

Comment (KingTerrytheTerrible) [reply code/__DEFINES/traits.dm]: i cant be fucked lol

File: hippiestation/code/game/objects/items/juggernaut.dm (hunk 2 above)
Comment (KingTerrytheTerrible) [reply hippiestation/code/game/objects/items/juggernaut.dm]: dropdel doesn't work in some situations

File: hippiestation/code/game/objects/items/juggernaut.dm (hunk 1 above)
Comment (KingTerrytheTerrible) [reply hippiestation/code/game/objects/items/juggernaut.dm]: done

**Comment by TrickyDickRichard** [comment]
ok +1 my ni- wait this is github

**Comment by KingTerrytheTerrible** [comment]
yep thats what happened to my account(s)

**Comment by KingTerrytheTerrible** [comment]
as for the video, https://gfycat.com/EagerAppropriateAmericanshorthair
the monkeys not being stunned is fixed

**Comment by QuarianCommando** [comment]
+1

**Comment by Neo-0** [comment]
so even when you drop it, will you still have these traits?

**Comment by steamp0rt** [comment]
@Neo-0 only way to drop it is to be beheaded

**Comment by Neo-0** [comment]
ah, nvm

**Comment by KingTerrytheTerrible** [comment]
pls merging

**Comment by TrickyDickRichard** [comment]
merge now

**Comment by zamolxius** [comment]
make it make you too ANGERY to use guns and able to throw ppl thru walls maybe

**Comment by KingTerrytheTerrible** [comment]
you cant use guns, i already did that. your hands are too big

**Comment by Manly-Dwarf** [comment]
Resolve the conflicts.

**Comment by Manly-Dwarf** [comment]
I'm expecting this to be horrible, but if it is then I can just move it to the surplus crate.

**Comment by KingTerrytheTerrible** [comment]
this was a bad idea, will balance shortly

**Comment by karmaisblackandbluepilled** [comment]
>30-50 damage a hit
Jesus Christ on the Cross what the hell.

**Comment by TrickyDickRichard** [comment]
its based it doesnt have to be balanced

**Comment by Manly-Dwarf** [comment]
ok it was funny but I'm removing this now

**Comment by Manly-Dwarf** [comment]
#12472 

i hate 20 TC traitor items so fucking much
but at least this was pretty funny at first

**Comment by MaterialisticThings** [comment]
Seems pretty fucking overpowered. How about giving you a limit of how many walls you can push through, every wall you push through you're damaged, same visibility as welding goggles? If we make this happen it needs to be 20 TC or be in a surplus crate because stun vulnerability is very powerful.

**Pull request merged**